qemu/target-arm/helper.c

```diff
@@ -1883,7 +1883,7 @@ static const ARMCPRegInfo v8_cp_reginfo[] = {
       ARM_CP_NOP, PL1_W },
     { "ICIALLU", 15,7,5, 0,0,0, 0,
       ARM_CP_NOP, PL1_W },
-    { "ICIMVAU", 15,7,5,0,1, 0,
+    { "ICIMVAU", 15,7,5, 0,0,1, 0,
```

Comment:
it seems this is the only place you fixed, but the rest jut got some accident changes, right?